data/brands/amenity/fast_food.json

```diff
@@ -8403,7 +8403,7 @@
         "brand:en": "McDonald's",
         "brand:ru": "МакДоналдс",
         "brand:uk": "МакДональдз",
-        "brand:wikidata": "Q38076",
+        "brand:wikidata": "Q4043856",
```

Comment:
McDonald’s in Ukraine is a business unit, but is it a brand distinct from the global McDonald’s brand? Wikidata doesn’t seem to think so.